lib/nx_signal.ex

```diff
@@ -257,7 +257,7 @@ defmodule NxSignal do
 
     as_windowed_parse_non_reflect_opts(
       shape,
-      Keyword.put(opts, :padding, [{div(window_length, 2), div(window_length, 2) - 1}])
+      Keyword.put(opts, :padding, [{div(window_length, 2), div(window_length, 2)}])
```

Comment:
@polvalente this changes some output shapes (the currently failing tests), but that's what `torch.stft` does. wdyt?

Reply:
Does it just decrease the number of windows?

Reply:
The opposite, it adds an extra window, where half of the window is padded.

Reply:
I think it's ok then